torch/_meta_registrations.py

```diff
@@ -407,6 +412,19 @@ def functional_sym_constrain_range(size, min, max, dep_token):
     return dep_token
 
 
+@register_meta(aten.sym_constrain_range_for_size.default)
+def sym_constrain_range_for_size(size, min=None, max=None):
+    if isinstance(size, (SymFloat, SymBool)):
+        raise ValueError("Constraining SymFloat or Symbool is nyi")
+    _constrain_range_for_size(size, min=min, max=max)
+
+
+@register_meta(aten._functional_sym_constrain_range_for_size.default)
+def functional_sym_constrain_range_for_size(size, min, max, dep_token):
+    aten.sym_constrain_range_for_size(size, min=min, max=max)
+    return dep_token
```

Comment:
Do we need to clone dep_token instead of returning itself directly?

Reply:
I think that is only true for Cpp kernels.

Reply:
No, you need to clone here too lol